openedx_events/tooling.py

```diff
@@ -80,7 +86,7 @@ def generate_signal_metadata(self):
             minorversion=self.minor_version,
         )
 
-    def send_event(self, send_robust=False, **kwargs):
+    def send_event(self, **kwargs):
```

Comment:
In case the above question is not true, do we really want to stop having this capability here?

Reply:
You're right, we shouldn't. I'll change this to: `send_robust=True,`